shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -348,9 +352,21 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
   val APPROX_PAR_META_DATA: Int = 10 * 1024 * 1024 // we are estimating 10MB
   val BYTES_ALLOWED_PER_BATCH: Long = _2GB - APPROX_PAR_META_DATA
 
+  private def getStatsFromCachedBatch(
+     cachedBatch: ParquetBufferConsumer,
+     schema: StructType,
+     conf: SQLConf): InternalRow = {
```

Comment:
nit: indenting seems slightly off

Reply:
I don't see where. Can you point out please?

Reply:
This code is 3-space indented instead of 4-space for continuations as spec'd in the style guidelines.  There are other occurrences of 3-space continuations in this file including in new methods added by this PR.

Reply:
Thanks, my idea was pulling settings from cudf-java. It probably got switched while working on cudf at some point. 

Is it OK if I reformat this file and create extra noise in the PR? I would prefer to do a separate PR to tackle the indentation

Reply:
Created draft https://github.com/NVIDIA/spark-rapids/pull/2538. I will update it once this PR is merged